tool/src/org/antlr/v4/codegen/target/DartTarget.java

```diff
@@ -38,10 +33,9 @@
 		"default", "get", "rethrow", "while",
 		"deferred", "hide", "return", "with",
 		"do", "if", "set", "yield",
-	};
 
-	/// Avoid grammar symbols in this set to prevent conflicts in gen'd code.
-	protected final Set<String> badWords = new HashSet<String>();
+		"rule", "parserRule"
```

Comment:
I see that this was copied from the previous targets but I'm wondering what the logic is...

Reply:
As well as I.